modules/sensor/instance.tf

```diff
@@ -0,0 +1,62 @@
+resource "google_compute_instance_template" "sensor_template" {
+  name         = "${var.name_prefix}-mig-template"
```

Comment:
The entire name should be a variable so folks can configure it to their own internal naming standards. Same for all the resources really.